nlp/src/main.py

```diff
@@ -54,4 +54,5 @@ def process_data(data):
 
 
 if __name__ == '__main__':
+    get_version()

```

Comment:
I think it'd be really useful to add some environment checks before starting the server.

* Are all the models from `models.version` downloaded and up to date?
* Are all the env variables set?

Start the server only if it can work correctly when the requests come in.

Reply:
Model checking take place in **update_model_version** function, it checks if files are available locally with current versions based on models file. The functions return True if the newest file already exists or when the newest version is downloaded successfully.